perfomance/README.md

```diff
@@ -1,20 +1,52 @@
 # Instructions
 
-Make sure you have a running instance of Hearth
+The k6 perfomance testing tool is used for these tests. The results of the tests are stored in influxdb and can be visualised on chronograf. These visualization tools and hearth have to be running for the tests to be executed. All these can be started using the following command:
 
-## Load Tests
+```bash
+docker-compose up -d
+```
 
-Test the number of concurrent users that can be handled by the Hearth under
-realistic circumstances.
+## Load Tests
 
-To perform a load test, substitute `<BASE_URL>` with url of the running Hearth instance and `<PATH>` with the resource path. Some resource paths are given in the table below
-run the following command:
+Test the number of concurrent users that can be handled by Hearth under
+realistic circumstances. To perform a load test, substitute `<BASE_URL>` with url of the running Hearth instance and `<PATH>` with the resource path and then run the following command:
 
 ```bash
-docker run -e 'BASE_URL=<BASE_URL>' -e 'RESOURCE_PATH=<PATH>'--network="host" -i -v $PWD:/src loadimpact/k6 run /src/load.js
+docker run -e 'BASE_URL=<Base_url>' -e 'PATH=<Path>' --network host -i -v $PWD:/src loadimpact/k6 run /src/load.js
+```
 
-| Resource     | PATH                        |
+The default base url is <http://localhost:3447> and 'fhir/Patient' for the path. Below are some examples of paths that can be used
+
+| Resource     | Path                        |
 | ------------ | --------------------------- |
-| Patient      | `/fhir/patient`             |
-| Practitioner | `/fhir/practitioner`        |
-| Location     | `/fhir/location`            |
+| Patient      | `/fhir/Patient`             |
+| Practitioner | `/fhir/Practitioner`        |
+| Location     | `/fhir/Location`            |
+
+## Volume Tests
+
+Test the throughput of Hearth when handlng large volumes. Substitute the base url and the path and run:
+
+```bash
+docker run -e 'BASE_URL=<Base_url>' -e 'PATH=<Path>' --network host -i -v $PWD:/src loadimpact/k6 run /src/volume.js
+```
+
+## Stress Tests
+
+Test the routing overhead of Hearth under maximum throughput. Substitute the base url and the path and run:
+
+```bash
+docker run -e 'BASE_URL=<Base_url>' -e 'PATH=<Path>' --network host -i -v $PWD:/src loadimpact/k6 run /src/volume.js
+```
+
+## InfluxDB Output
+
+Ensure that influxdb and chronograf docker containers are running. Once they are up and running you can access Chronograf at http://localhost:8888. This is if the doccker containers are hosted on a local machine. If not the ip address will be different but the ports are the same (same applies for the influxdb url).
```

Comment:
We can make the URL for chronograf a link here as the user might want to click  it and see the web page

Reply:
Typo: doccker